tests/test_universal_word_final_hardening.py

```
@@ -0,0 +1,155 @@
+    calls = {"draft": 0, "repair": 0}
+
+    class FakeService:
+        async def draft(self) -> dict:
+            calls["draft"] += 1
+            # Simulate the client opening another document while the initial
+            # model draft is in flight. Any bounded repair must now be aborted.
+            state.data["request_kind"] = "contract" if kind != "contract" else "claim"
+            state.data["request_id"] = "request-new"
+            return await self._quality_repair(current_payload={"safe": True})
+
+        async def _quality_repair(self, **_kwargs) -> dict:
```

Ruff (0.16.2)

[warning] Missing type annotation for `**_kwargs`

(ANN003)